lib/seattleflu/db/cli/command/__init__.py

```diff
@@ -9,3 +10,18 @@
     "clinical",
     "user",
 ]
+
+def trim_whitespace(df: pd.DataFrame) -> pd.DataFrame:
+    """ Trims leading and trailing whitespace from strings in *df* """
+    str_columns = df.columns[every_value_is_str_or_na(df)]
+    df[str_columns] = df[str_columns].apply(lambda column: column.str.strip())
+
+    return df
+
+
+def every_value_is_str_or_na(df):
+    """
+    Evaluates whether every value in the columns of a given DataFrame *df* is
+    either a string or NA.
+    """
+    return df.applymap(lambda col: isinstance(col, str) or pd.isna(col)).all()
```

Comment:
I would prefer to keep these functions local to the code using them until we have a need to use them elsewhere.